services/frontend-v3/src/pages/admin/Competency.jsx

```diff
@@ -0,0 +1,22 @@
+// import React, { Component } from "react";
+// import AdminOptionEdit from "../components/admin/AdminOptionEdit";
```

Comment:
why is it all commented out? should we just delete it?

Reply:
Deleted